pkg/README.md

```diff
@@ -46,11 +47,8 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
    kubectl apply -f ./manifests/gateway/gateway.yaml
    ```
    > **_NOTE:_** This file couples together the gateway infra and the HTTPRoute infra for a convenient, quick startup. Creating additional/different InferencePools on the same gateway will require an additional set of: `Backend`, `HTTPRoute`, the resources included in the `./manifests/gateway/ext-proc.yaml` file, and an additional `./manifests/gateway/patch_policy.yaml` file. ***Should you choose to experiment, familiarity with xDS and Envoy are very useful.***
-   
-   
-
 
-1. **Deploy Ext-Proc**
+1. **Deploy the Inference Extension and InferencePool**
```

Comment:
As I mentioned above, the InferencePool is now bundled with the ext-proc since the ext-proc is specifically configured for this pool.